engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -335,10 +336,12 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
       finalizeWorkflow(data, workflowDescriptor, jobKeys, CallOutputs.empty, Option(List(failures)))
     case Event(WorkflowExecutionAbortedResponse(jobKeys),
     data @ WorkflowActorData(_, Some(workflowDescriptor), _, _, _)) =>
-      finalizeWorkflow(data, workflowDescriptor, jobKeys, CallOutputs.empty, None)
+      finalizeWorkflow(data, workflowDescriptor, jobKeys, CallOutputs.empty, data.lastStateReached.failures)
```

Comment:
presumably we could fetch `lastStateReached` in the `data @ WorkflowActorData(...)` match?